deploy.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+FILEPATH=$1
+ENV=$2
+SHOP=$3
+
+gcloud container clusters get-credentials --project="prestashop-ready-$ENV" --zone="europe-west1-d" "prestashop-ready-cluster"
```

Comment:
This file need to be remove, it's useless :thinking:

Reply:
Oops yes